server/server.js

```diff
@@ -92,23 +89,21 @@ let startTurn = (gameId) => {
     }
 
     const current_index = games[gameId]['current_player'];
-    console.log('current_index:', current_index)
     const clientId = games[gameId]['clients'][current_index]
-    console.log('clientId:', clientId)
     const name = clients[clientId]['name']
-    console.log('name:', name)
-
 
     const randomWordsList = randomWords(3);
     const payload = {
-        'method': 'TURN',
-        'words': randomWordsList
+        'method': events.TURN,
+        'words': randomWordsList,
+        'turnTime':games[gameId]['turnTime'],
+        'gameTime':games[gameId]['gameTime']
     }
```

Comment:
turn time is needed to display the timer, but what about gameTime why it is needed on the client side ?